doc/classes/Physics2DShapeQueryResult.xml

```diff
@@ -27,6 +31,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns the ID of the [Object] that intersected with the shape at index [code]idx[/code].
```

Comment:
Is the object ID the same as the one returned by `Object.get_instance_id()`? I haven't had the time to check this.

Reply:
Seems so:
```
servers/physics_2d/space_2d_sw.cpp
93:             r_results[cc].collider_id = col_obj->get_instance_id();
94:             if (r_results[cc].collider_id != 0)
95:                     r_results[cc].collider = ObjectDB::get_instance(r_results[cc].collider_id);
184:    r_result.collider_id = res_obj->get_instance_id();
185:    if (r_result.collider_id != 0)
186:            r_result.collider = ObjectDB::get_instance(r_result.collider_id);
```